src/Common/Microsoft.MSBuildCache.Common.csproj

```diff
@@ -25,6 +25,7 @@
   </ItemGroup>
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.MSBuildCache.Common.Tests" />
+    <InternalsVisibleTo Include="Microsoft.MSBuildCache.AzurePipelines" />
```

Comment:
This should be avoided. Our plugin impls should not be "special" in any way.

Just make whatever you need public instead.